sbt_common/internal_model/src/test/scala/uk/ac/wellcome/models/work/test/util/WorksGenerators.scala

```diff
@@ -102,6 +102,9 @@ trait WorksGenerators extends ItemsGenerators {
       version = version
     )
 
+  def createUnidentifiedWorkOfRandomType: UnidentifiedWork =
```

Comment:
And here the use of "random" is plain wrong – createSourceIdentifier _always_ creates a Miro identifier.

Reply:
oops, thanks.  Yes it *was* really random but then I changed it back (minus the name obvs).  I'm in two minds about whether it should be random.